libs/features/georoutingv2/georoutingv2.js

```diff
@@ -236,26 +236,30 @@ function buildContent(currentPage, locale, geoData, locales) {
 
 async function getDetails(currentPage, localeMatches, geoData) {
   const availableLocales = await getAvailableLocales(localeMatches);
-  if (availableLocales.length > 0) {
-    const georoutingWrapper = createTag('div', { class: 'georouting-wrapper fragment' });
-    currentPage.url = window.location.hash ? document.location.href : '#';
-    if (availableLocales.length === 1) {
-      const content = buildContent(currentPage, availableLocales[0], geoData);
-      georoutingWrapper.appendChild(content);
-      return georoutingWrapper;
-    }
-    const sortedLocales = availableLocales.sort((a, b) => a.languageOrder - b.languageOrder);
-    const tabsContainer = createTabsContainer(sortedLocales.map((l) => l.language));
-    georoutingWrapper.appendChild(tabsContainer);
-
-    sortedLocales.forEach((locale) => {
-      const content = buildContent(currentPage, locale, geoData, sortedLocales);
-      const tab = createTab(content, locale.language);
-      georoutingWrapper.appendChild(tab);
-    });
+  if (!availableLocales.length) return null;
+  const { innerWidth } = window;
+  let svg = null;
+  if (innerWidth < 480) {
+    const { default: getMobileBg } = await import('./getMobileBg.js');
+    svg = getMobileBg();
+  }
+  const georoutingWrapper = createTag('div', { class: 'georouting-wrapper fragment', style: 'display:none;' }, svg);
```

Comment:
To address @honstar's comment - you could add a class that you could toggle, handling the `display` value from CSS to avoid the `!important`

Reply:
agree 👍

Reply:
A class would defeat the purpose. Let me explain:

The modal content (including the svg) gets added to the DOM before the css rules have been processed, creating a split second where the modal is unstyled (picture on left).
<img width="428" alt="Screenshot 2024-02-28 at 12 06 05 PM" src="https://github.com/adobecom/milo/assets/19690988/e4bea2fc-f361-4640-af58-4ccba5765294">
This creates CLS. To clarify, there was CLS before my changes, but because the inline svg is so tall, it increased the amount of CLS.

To prevent this, I'm hiding the modal until the stylesheet is loaded. That's why I have the `display: none;` inline and the `display: block !important;` in the css file. This means that the modal content is only seen once the styles are in place, preventing CLS.

There are, of course, other possible solutions to make sure the CSS loads before the modal is added to the DOM, but this method seemed simpler, would change less code, and wouldn't block the modal rendering while waiting for the styles.

Additionally, the reason why using `!important` is bad practice is because highjacks the normal inheritance structure and makes it messy to override the property downstream. However, I don't think is really a problem in our case. The `georouting-wrapper` is a very specific element. Are we really going to be creating different versions of it that will need different `display` values?

This was my thought process. If you still object, I can look at preventing the CLS another way. Let me know, thanks

Reply:
That's really interesting . I can't even reproduce that CLS no matter how much I throttle on my machine 👍 
I would have probably just waited for the css to finish processing then in the showModal method like so:
```
let resolve;
const cssProcessed = new Promise((resolveFunc) => resolve = resolveFunc);
const promises = [
  tabs ? loadBlock(tabs) : null,
  tabs ? loadStyle(`${miloLibs || codeRoot}/blocks/section-metadata/section-metadata.css`) : null,
  loadStyle(`${miloLibs || codeRoot}/features/georoutingv2/georoutingv2.css`, () => resolve()),
  cssProcessed,
];
await Promise.all(promises);
```
I think this is slightly better since we're avoiding an !important and you're waiting for the precise thing that you really wanted, but I'll leave it in your court if you really want to use. I'm not even sure it properly works, since I couldn't reproduce the cls ! :)

Reply:
I tried waiting for it to resolve ([see commit](https://github.com/adobecom/milo/pull/1916/commits/52c9fec7416df0fe5c6a8a76f6b966be758d5049)) and it added 0.222 CLS, pushing it into yellow territory. Given that, I'd prefer to keep the code as is. Please let me know if you still consider this a blocking issue. Thanks!

Reply:
ok, thanks for trying. 🙌 
All good from my side then.